tutorials/training_ppi.ipynb

```diff
@@ -0,0 +1,6991 @@
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:deeprankcore.dataset:\n",
+      "Checking dataset Integrity...\n",
+      "INFO:deeprankcore.dataset:Target classes set up to: [0, 1]\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "\n",
+      "Loading test data...\n",
+      "   ['data/ppi/processed/residue/proc-28835.hdf5', 'data/ppi/processed/residue/proc-28842.hdf5', 'data/ppi/processed/residue/proc-28839.hdf5', 'data/ppi/processed/residue/proc-28838.hdf5', 'data/ppi/processed/residue/proc-28843.hdf5', 'data/ppi/processed/residue/proc-28834.hdf5', 'data/ppi/processed/residue/proc-28837.hdf5', 'data/ppi/processed/residue/proc-28840.hdf5', 'data/ppi/processed/residue/proc-28841.hdf5', 'data/ppi/processed/residue/proc-28836.hdf5'] dataset                 : 100%|██████████| 10/10 [00:00<00:00, 894.69it/s, entry_name=proc-28836.hdf5]\n"
+     ]
+    }
+   ],
+   "source": [
+    "target = \"binary\"\n",
+    "task = \"classif\"\n",
+    "\n",
+    "print('Loading training data...')\n",
+    "dataset_train = GridDataset(\n",
+    "    hdf5_path = input_data_path,\n",
+    "    subset = list(df_train.entry), # selects only data points with ids in df_train.entry\n",
+    "    target = target,\n",
+    "    task = task\n",
+    ")\n",
+    "print('\\nLoading validation data...')\n",
+    "dataset_val = GridDataset(\n",
+    "    hdf5_path = input_data_path,\n",
+    "    subset = list(df_valid.entry), # selects only data points with ids in df_valid.entry\n",
+    "    target = target,\n",
+    "    task = task\n",
+    ")\n",
+    "print('\\nLoading test data...')\n",
+    "dataset_test = GridDataset(\n",
+    "    hdf5_path = input_data_path,\n",
+    "    subset = list(df_test.entry), # selects only data points with ids in df_test.entry\n",
+    "    target = target,\n",
+    "    task = task\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Trainer\n",
+    "\n",
+    "As for graphs, the class `Trainer` is used for training, validation and testing of the PyTorch-based CNN. "
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- Also in this case, `neuralnet` can be whatever neural network class which inherits from `torch.nn.Module`, and it shouldn't be specific to regression or classification in terms of output shape. Here we use `CnnClassification` (in `deeprankcore.neuralnets.cnn.model3d`), whose architecture is shown below:\n",
```

Comment:
see my comment on grid `Trainer` class. 

Maybe add a comment about the optimizer, similar to grid case?